kelpie-f446/Core/Modules/InterCommsModule.h

```diff
@@ -8,6 +8,18 @@
 #ifndef MODULES_INTERCOMMSMODULE_H_
 #define MODULES_INTERCOMMSMODULE_H_
 
+#include "stm32f4xx_hal.h"
+#include <stdint.h>
+#include "UserTypes.h"
 
+typedef struct ICommsMsg_t{
+	uint8_t msgId;
+	uint8_t dataLen;
```

Comment:
uint16_t same reason

Reply:
I also made this fix in UserTypes.h since it also used uint8_t